azurerm/internal/services/network/subnet_resource_test.go

```diff
@@ -47,6 +47,50 @@ func TestAccSubnet_basic_addressPrefixes(t *testing.T) {
 	})
 }
 
+func TestAccSubnet_complete_addressPrefixes(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_subnet", "test")
+	r := SubnetResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.complete_addressPrefixes(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
```

Comment:
this test is superfluous since it's included in the first step below?

Reply:
This is for the sake of completeness and self-contained test.